packages/common-lib/src/services/subjectListRegistryService.js

```diff
@@ -21,6 +21,8 @@ export const getProgramId = async (props) => {
     { headers }
   )
   if (programID?.data?.data) {
+    console.log(programID?.data?.data[0]?.programId)
```

Comment:
@Aar-if console remove
Program Id we can keep as it is in local storage as it is same till the session close & this year